priv/repo/migrations/20220915094102_add_foreign_references_and_delete_behaviors_to_tables.exs

```diff
@@ -25,33 +23,35 @@ defmodule Ret.Repo.Migrations.AddForeignReferencesAndDeleteBehaviorToTables do
 
   def up do
     for {table, foreign_table, column, on_delete_action} <- @missing_references do
-      add_foreign_reference(table, column, foreign_table, column, on_delete: on_delete_action)
+      alter table(table) do
+        modify(column, references(foreign_table, column: column, on_delete: on_delete_action))
+      end
     end
 
     for {table, column, foreign_table, foreign_column, on_delete_action} <- @missing_on_delete_action do
-      execute_drop_foreign_constraint(table, column)
-      add_foreign_reference(table, column, foreign_table, foreign_column, on_delete: on_delete_action)
+      drop_foreign_constraint(table, column)
+
+      alter table(table) do
+        modify(column, references(foreign_table, column: foreign_column, on_delete: on_delete_action))
+      end
     end
   end
 
   def down do
     for {table, _foreign_table, column, _on_delete_action} <- @missing_references do
-      execute_drop_foreign_constraint(table, column)
+      drop_foreign_constraint(table, column)
     end
 
     for {table, column, foreign_table, foreign_column, _on_delete_action} <- @missing_on_delete_action do
-      execute_drop_foreign_constraint(table, column)
-      add_foreign_reference(table, column, foreign_table, foreign_column, on_delete: :nothing)
-    end
-  end
+      drop_foreign_constraint(table, column)
```

Comment:
on `drop()` function, shouldn't it do the opposite of `up()` This looks to be the same as `up()`